### PR TITLE
Refactor statistics

### PR DESCRIPTION
**Pull request by hadriensevel**
Refactor the statistics.js file

### CODE REVIEW

**Comment by giacomoorsi** [comment]
Just to make sure, did you test it or did you just refactor the code offline?

File: website/js/statistics.js [hunk 1]
```diff
@@ -2,125 +2,51 @@
...
-    return data;
-});
-
-const promise_stop_data = d3.csv("data/data_stop.csv").then(function (data) {
-    return data;
-});
-
 function pascalize(str) {
-    return str.replace(/(\w)(\w*)/g, function (g0, g1, g2) { return g1.toUpperCase() + g2.toLowerCase(); });
+    return str.replace(/\w+/g, word => word.charAt(0).toUpperCase() + word.slice(1).toLowerCase());
```
Comment (giacomoorsi) [inline website/js/statistics.js:6]: Why the change here? Are we 100% sure it has the same behavior?

**Comment by hadriensevel** [comment]
I tested it

File: website/js/statistics.js (hunk 1 above)
Comment (hadriensevel) [reply website/js/statistics.js:6]: Yes it's exactly the same behaviour, except that the regular expression is simpler and the function easier to understand.

**Review by giacomoorsi — APPROVED** [review]
